ErgoScript-Developer-Course/batch-0/ergonaut_alpha/ErgoscriptHW2/ErgoscriptHW2.scala

```diff
@@ -0,0 +1,32 @@
+val bools: = True
+val wrong: = False
+
+if(true){
+    val num + 1
+} else {num - 1}
```

Comment:
you have to return a value. Therefore it would be something along the line of 

if (true) {
    val num = 1
    num // <- this returns the value.
} else {
    val num = 01
    num // <- you still have to return it
}